cognite/client/data_classes/data_modeling/views.py

```diff
@@ -863,3 +865,21 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
             output["connection_type"] = "multi_reverse_direct_relation"
 
         return output
+
+
+@dataclass
+class TargetUnit(CogniteObject):
```

Comment:
Seems kinda randomly placed in this module? It's only used in InstancesAPI, so I'd suggest putting it in data_classes/instances.py